fastchat/serve/shutdown_serve_by_shell.py

```diff
@@ -0,0 +1,31 @@
+"""
+Usage：
+python llm_api_shutdown.py --serve all
+options: "all","controller","model_worker","openai_api_server"， `all` means to stop all related serves 
+"""
+import sys
+import os
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+import subprocess
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--serve", choices=["all", "controller", "model_worker", "openai_api_server"]
```

Comment:
`--serve` may not sound that intuitive. Is `--down` better?

Reply:
OK